ts/src/pro/bingx.ts

```diff
@@ -753,33 +753,47 @@ export default class bingx extends bingxRest {
         //        ]
         //    }
         //
-        const data = this.safeValue (message, 'data', []);
```

Comment:
Unfortunately, this fix is wrong. For spot market, 'data' is an object. Using safeList makes data an empty array.

Reply:
@CH-EdwinB we will take a look